Tasks/DownloadPackageV1/task.json

```diff
@@ -0,0 +1,175 @@
+{
+    "id": "8d6e8f7e-267d-442d-8c92-1f586864c62f",
+    "name": "DownloadPackage",
+    "friendlyName": "Download Package",
+    "description": "Download a package from a Package Management feed in Azure Artifacts or TFS. \r\n Requires the Package Management extension.",
+    "helpUrl": "",
+    "helpMarkDown": "Needs Package Management extension to be installed",
+    "category": "Utility",
+    "author": "ms-vscs-rm",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 18
+    },
+    "demands": [],
+    "minimumAgentVersion": "1.99.0",
+    "inputs": [
+        {
+            "name": "packageType",
+            "type": "pickList",
+            "label": "Package Type",
+            "required": true,
+            "helpMarkDown": "Select the package type",
+            "defaultValue": "Microsoft.Insights/components",
+            "options": {
+                "Maven": "Maven",
+                "Npm": "Npm",
+                "NuGet": "NuGet",
+                "Python": "Python",
+                "Universal": "Universal"
+            }
+        },
+        {
+            "name": "feed",
+            "type": "pickList",
+            "label": "Feed",
+            "defaultValue": "",
+            "required": true,
+            "properties": {
+                "EditableOptions": "True"
+            },
+            "helpMarkDown": "Select the feed"
+        },
+        {
+            "name": "view",
+            "type": "pickList",
+            "label": "View",
+            "defaultValue": "",
+            "required": true,
```

Comment:
this should be optional, and hidden if a protocol that doesn't contain views is selected.
I just tried it out, setting default value to " " seems to make the update logic work.

The dataSourceBindings then need to be updated like {{{feed}}}{{#if view}}@{{{view}}}{{/if}}